### ISSUE TITLE
Distribute work (Slurm, AirFlow)

### ISSUE THREAD

**Issue by cbizon** [NCATSTranslator/Babel#35]
snakemake workflows can be run in parallel.   It can interface with slurm and AirFlow can also handle it.  What's the right implementation at RENCI?

**Comment by gaurav** [comment]
At the moment, we run Babel on a single Kubernetes node with 500G of memory and several CPUs. This takes about a day to complete, which is fast enough that it hasn't been a priority to replace it with something faster, so I'll mark this as low priority but leave this issue open. If future features increase the processing time dramatically, we can re-evaluate our options.

**Comment by gaurav** [comment]
Snakemake supports Slurm (https://snakemake.readthedocs.io/en/v7.19.1/executing/cluster.html) and I've increased the amount of disk space we have on Hatteras, so all the Babel developers should be able to run Babel on Hatteras using Slurm using different directories in the `/projects/babel` space. One concern is that Babel is made up of a lot of quick-running tasks, so the work of starting up distinct Slurm jobs for each of them might slow the whole process down significantly. But hopefully it won't slow things down too much -- we won't know until we try it!
